### ISSUE TITLE
Question about the evaluation of part seg 

### ISSUE THREAD

**Issue by LiDaiY** [yanx27/Pointnet_Pointnet2_pytorch#53]
Hi，thanks for your code， it's really nice.
I have one question about the the  evaluation method of part seg. I noticed in the `111` line of  `test_partseg.py`, you calc the acc by following code:

> cur_pred_val[i, :] = np.argmax(logits[:, seg_classes[cat]], 1) + seg_classes[cat][0]

This assumes that, during test we already know which parts each category contains. I think it may reduce the difficulty during test.  Is this implementation the same as official?

**Comment by yanx27** [comment]
Hi @LiDaiY, thanks for your attention. Official pointnet++ gives two options to train part segmentation, which are train for each category separately and train with one-hot. I just implement the latter one, and for each sample we already know its category, which is a fair comparison. This test codes also modify by official pointnet++ (see [here](https://github.com/charlesq34/pointnet2/blob/42926632a3c33461aebfbee2d829098b30a23aaa/part_seg/train.py#L280)). Hope this can help you :)

**Comment by LiDaiY** [comment]
Hi, @yanx27 , thanks for your prompt and detailed reply, it helps me a lot.
